linux_os/guide/system/permissions/mounting/kernel_module_cramfs_disabled/rule.yml

```diff
@@ -27,9 +27,7 @@ identifiers:
     cce@rhcos4: CCE-82514-1
 
 references:
-    cis@rhel8: 1.1.1.1
-    cis@sle15: 1.1.1.1
-    cis@ubuntu1804: 1.1.1.1
+    cis: 1.1.1.1
```

Comment:
cramfs remains the same, but 1.1.1.x change between 18.04 and 20.04, so I'm inclined to leave these separate.